src/rgw/rgw_data_sync.cc

```diff
@@ -1942,28 +1942,38 @@ RGWCoroutine *RGWRemoteBucketLog::init_sync_status_cr()
   return new RGWInitBucketShardSyncStatusCoroutine(&sync_env, bs, init_status);
 }
 
+#define BUCKET_SYNC_ATTR_PREFIX RGW_ATTR_PREFIX "bucket-sync."
```

Comment:
with the raw_attrs changes, is there a reason we still need to add a prefix and mess with backward compatibility? it looks like most/all other uses of RGW_ATTR_PREFIX are related to head objects, so it seems useful to keep that distinction

Reply:
In general any attrs that rgw sets should be prefixed, and that's in order to distinct these from attrs that are being set on the same objects by objclass methods (e.g., cls_lock, etc.)